lib/solveEquation/stepThrough.js

```diff
@@ -183,11 +221,18 @@ function addSimplificationSteps(steps, equation, debug=false) {
   let oldEquation = equation.clone();
 
   const leftSteps = simplifyExpressionNode(equation.leftNode, false);
+  const leftFactorSteps = factor(equation.leftNode, false);
   const leftSubSteps = [];
   for (let i = 0; i < leftSteps.length; i++) {
     const step = leftSteps[i];
     leftSubSteps.push(EquationStatus.addLeftStep(equation, step));
   }
+
+  for (let i = 0; i < leftFactorSteps.length; i++) {
```

Comment:
`leftFactorSteps.each(step => leftSubSteps.push(EquationStatus.addLeftStep(equation, step)))`

I'm a bit confused - why are there left/right Steps as well as left/right FactorSteps? What's an example that would have steps coming from both?